screen/edit-transaction/src/main/java/com/ivy/transaction/EditTransactionViewModel.kt

```diff
@@ -706,7 +709,10 @@ class EditTransactionViewModel @Inject constructor(
                     accountsChanged = false
                 }
 
-                transactionWriter.save(loadedTransaction().toEntity())
+                val trans = with(transactionMapper) {
+                    loadedTransaction().toEntity().toDomain()
```

Comment:
Why do we do this? Can't we just pass the domain object?

Reply:
Well, then i need to create toDomain directly from LegacyTransaction ,as currently from TransactionEntity only it is possible, that too is there in Mapper.
but yeah reverse is implemented ( transaction to LegacyTransaction )